src/actions/index.js

```diff
@@ -11,8 +11,17 @@ export const DECREMENT = 'DECREMENT';
 
 export const increment = () => {
   // Fill in this function
+  return {
+    type: INCREMENT,
+    payload: +1
+  }
```

Comment:
Interesting you are actually sending a payload back from here. For this particular problem this could also be done just by passing the action type and handling the logic in your reducer without the need of a payload.